frontend/packages/dev-console/src/components/health-checks/health-check-probe-utils.ts

```diff
@@ -0,0 +1,100 @@
+import { HealthChecksProbeType, ProbeType } from './health-checks-types';
+
+export const getHealthChecksProbe = (probe: string) => {
+  switch (probe) {
+    case HealthChecksProbeType.ReadinessProbe: {
+      return {
+        formTitle: 'Readiness Probe',
+        formSubtitle:
+          "A Readiness probe checks if the container is ready to handle requests. A failed readiness probe means that a container should not receive any traffic from a proxy, even if it's running.",
+      };
+    }
+    case HealthChecksProbeType.LivenessProbe: {
+      return {
+        formTitle: 'Liveness Probe',
+        formSubtitle:
+          'A Liveness probe checks if the container is still running. If the liveness probe fails the container is killed.',
+      };
+    }
+    case HealthChecksProbeType.StartupProbe: {
+      return {
+        formTitle: 'Startup Probe',
+        formSubtitle:
+          'A Startup probe checks if the application within the container is started. If the startup probe fails the container is killed.',
+      };
+    }
+    default:
+      return undefined;
+  }
+};
+
+export const healthChecksProbeInitialData = {
+  healthChecks: {
```

Comment:
Again, why do you have an object with three probe types as property but having same data as values for all of these properties?